src/d3a/models/strategy/load_hours.py

```diff
@@ -109,6 +107,9 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None,
 
     @property
     def active_markets(self):
+        # If use_market_maker_rate is true, overwrite final_buying_rate to market maker rate
```

Comment:
Why is it needed here ? It should suffice to be only in the event_activate method

Reply:
OMG, you are totally right! I was too focused on the word `activate` and confused active_markets with event_activate. Thanks for catching this!